### DISCUSSION TITLE
On the open source grants jury

### DISCUSSION THREAD

**Discussion by knadh** [github/india#66] [Open Source] [answered]
Happy to see the [GitHub grants](https://github.com/github/india/blob/main/Open%20Source/GRANTS.md) program. Looking forward to Indian FOSS projects coming forward and applying.

One thing that came up in a [community discussion](https://tg.fossunited.org/2021-03_6.html#2021-03-26) was that there are no developers, FOSS maintainers, or people from the broader FOSS community in the jury. It would be great if the jury included people associated with FOSS. Maybe the community here can put forward recommendations.

**Comment by msupernaut** [comment]
Great feedback @knadh  ! 

Who do you think could be a couple of nominations from the community? 

Since I assumed many of the contributors & maintainers will likely be the recipients of this Grant, i hadn’t got anyone in the evaluation team as it would become a conflict for them. 

Open to suggestions as is always the case of working in Public ! 

Let me know the community’s thoughts. 

Who do you think can be a neutral representative from our community ?

**Comment by knadh** [comment]
Thanks @msupernaut. Personal suggestions (checked with them whether they're okay to be nominated):

- @anandology - Anand Chithpothu. FOSS developer and a long time community participant and technical trainer. He leads community efforts and is experimenting with a code learning platform at @fossunited.
- @jace - Kiran Jonnalagadda. FOSS developer, founder @HasGeek. Connected to a large number of FOSS groups in India.

**Comment by msupernaut** [comment, marked as answer]
hey @knadh and everyone ! 

Excited to share with you that we have @janakiramm as our community representative in our Open Source grant jury. 

Janakiram is an independent architect and author who is well known in the developer community

He has a breadth and a depth understanding of various technologies - both proprietary as well as open source and has worked extensively with Google, Microsoft, Amazon, IBM, DigitalOcean, VMWare and other companies as an advisor and consultant.

We are working with the Jury members over next 2 weeks and would be announcing the recipients soon.

**Reply by jace** [reply]
@janakiramm is a great pick!

**Reply by ohbus** [reply]
@msupernaut Are there any updates about who received the grants?

**Reply by msupernaut** [reply]
> @msupernaut Are there any updates about who received the grants?

Yes @ohbus , we just announced this on 13/Sep(Programmer's day) : Please view https://github.blog/2021-09-12-recipients-open-source-grants-github-sponsors-india/

**Comment by msupernaut** [comment]
hi Everyone ! !

Excited to share that we have our first set of 15 recipients for the GitHub Grant for Open Source. We have also opened GitHub Sponsors to select developers in India so that they can get financial support from the community at large. 

Feel free to read more in a [blog I posted ](https://github.blog/2021-09-12-recipients-open-source-grants-github-sponsors-india/)this week.

👀 @knadh @jace
